packages/minizip.rb

```diff
@@ -1,40 +1,34 @@
 require 'package'
+require_relative 'zlibpkg'
 
 class Minizip < Package
   description 'Minizip is a simple package to zip/unzip files, from zlib.'
-  homepage 'https://www.zlib.net/'
-  @_ver = '1.2.11'
+  homepage Zlibpkg.homepage
```

Comment:
Not a valid url?

Reply:
it is a valid url...

Reply:
Doesn't resolve for me.  You are confusing me.

Reply:
Doesn't resolve for me.  You are confusing me.

Reply:
Oh! `homepage` is a property of the class `Zlibpkg` which i've required in the line `require_relative zlibpkg` at the top of the file. Because `zlibpkg` and `minizip` all use the same source, i've dried up the code a bit and made future updates easier by using class properties, e.g., chromebrew will interpret `Zlibpkg,homepage` as the `homepage` parameter in the `Zlibpkg` class. The same goes for `Zlibpkg.source_url` and `Zlibpkg.source_sha256`.

Reply:
Doesn't that overcomplicate things?  You still need to manage the versions in property values.

Reply:
doesn't having to copy and paste things around overcomplicate things? also minizip and zlib use the same tarball so naturally the versions will be the same.

Reply:
@uberhacker: @saltedcoffii is reusing the defined `homepage` and `source_{url,sha256}` from `zlibpkg.rb` (thats why `require_relative 'zlibpkg'` in line 2) and access it with the class const

Reply:
I gathered but what happens when we use `crew search zlibpkg -v`?  Is it going to display the correct homepage, version, etc. properties.  These are the kinds of things that have to be considered before willy-dilly doing something this drastic.

Reply:
yes it does display the correct things.